slack.json

```diff
@@ -0,0 +1,21 @@
+{
+  "runtime": "deno",
+  "hooks": {
+    "build": "deno run -q --unstable --import-map=import_map.json --allow-read --allow-write --allow-net https://deno.land/x/deno_slack_builder@0.0.5/mod.ts",
+    "get-manifest": "deno run -q --unstable --import-map=import_map.json --allow-read --allow-net https://deno.land/x/deno_slack_builder@0.0.5/mod.ts --manifest",
+    "get-hooks": "deno run -q --unstable --import-map=import_map.json --allow-read --allow-net https://raw.githubusercontent.com/slackapi/deno-slack-builder/get-scripts/src/get-scripts.ts",
```

Comment:
This seems like an outlier — we have a [deno-slack-hooks](https://github.com/slackapi/deno-slack-hooks) repo — should we wait to implemented `get-hooks` until we can set this up and use it instead?

Also from my understanding when we have `get-hooks` implemented it should be the only hook in the `slack.json` file and the others should be provided by a pre-fetch call to `get-hooks`? cc @filmaj

Reply:
Yeah a couple of things in here would need changing:

- a single `get-scripts` (not `get-hooks`) under the `hooks` property, otherwise, the other hooks defined in this file like `build`, `get-manifest`, `start` override / take precedence over what is returned in `get-scripts` ([as described in Michael's PR implementing `get-scripts`](https://github.com/slackapi/slack-cli/pull/476), he goes into the scenario in the last scenario laid out in the original post of the PR)
- `get-scripts` should point to the `deno-slack-hooks` repo, yep

I was personally going to wait until the dust settles on the other PRs in the other projects as it's definitely a moving target for now.

Reply:
@filmaj I forgot to ask this on @mwbrooks's PR about `get-hooks` vs. `get-scripts`. 

I think we should alignin on `hooks` and `get-hooks` instead of `hooks` and `get-scripts`. Since there is also a `scripts` section in this PR (which is where devs could add other scripts specifically for their project such as `test`).

Reply:
Oh ok. Let's get all the changes into the official specification document and then we can do the needful and update all the repos with all the things.

Reply:
Sounds like until the above is ready we should remove `get-hooks` from this PR?

Reply:
> * ... the other hooks defined in this file like `build`, `get-manifest`, `start` override / take precedence over what is returned in `get-scripts` ([as described in Michael's PR implementing `get-scripts`](https://github.com/slackapi/slack-cli/pull/476), he goes into the scenario in the last scenario laid out in the original post of the PR)
> * `get-scripts` should point to the `deno-slack-hooks` repo, yep
> 
> I was personally going to wait until the dust settles on the other PRs in the other projects before looking at this as it's definitely a moving target for now.

@filmaj I consider these comments the blockers for including `get-hooks`

Reply:
Ah yeah that makes sense!

Reply:
Yep, I think PR https://github.com/slackapi/slack-cli/pull/477 is changing from `get-scripts` to `get-hooks` because all hooks are now under the `hooks: {}` object.

> Also from my understanding when we have get-hooks implemented it should be the only hook in the slack.json file and the others should be provided by a pre-fetch call to get-hooks?

Correct, only very rare cases would there be both a `get-hooks` and other hook scripts. For our template, we should use `get-hooks` once the `deno-slack-hooks` repo is ready. Until then, we should not use `get-hooks`.

Reply:
should i remove `get-hooks` from this template?

Reply:
https://github.com/slackapi/deno-slack-hooks/pull/3 PR here?

Reply:
i removed get-hooks until further notice